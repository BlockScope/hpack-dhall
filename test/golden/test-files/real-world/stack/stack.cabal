cabal-version: 1.24

-- This file has been generated from stack.dhall by hpack version 0.31.0.
--
-- see: https://github.com/sol/hpack
--
-- hash: 35546ab238db21c4065ad4b710944992f1a8c40e8372fc8c8b35b8a730f40e3a

name:           stack
version:        1.10.0
synopsis:       The Haskell Tool Stack
description:    Please see the README.md for usage information, and
                the wiki on Github for more details.  Also, note that
                the API for the library is not currently stable, and may
                change significantly, even between minor releases. It is
                currently only intended for use by the executable.
category:       Development
homepage:       http://haskellstack.org
bug-reports:    https://github.com/commercialhaskell/stack/issues
author:         Commercial Haskell SIG
maintainer:     manny@fpcomplete.com
license:        BSD3
build-type:     Custom

source-repository head
  type: git
  location: https://github.com/commercialhaskell/stack

custom-setup
  setup-depends:
      Cabal >=2.4
    , base >=4.10 && <5
    , filepath
