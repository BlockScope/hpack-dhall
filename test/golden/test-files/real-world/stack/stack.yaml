name: stack
version: 1.10.0
synopsis: The Haskell Tool Stack
description: ! 'Please see the README.md for usage information, and

  the wiki on Github for more details.  Also, note that

  the API for the library is not currently stable, and may

  change significantly, even between minor releases. It is

  currently only intended for use by the executable.'
category: Development
homepage: http://haskellstack.org
author: Commercial Haskell SIG
maintainer: manny@fpcomplete.com
license: BSD3
github: commercialhaskell/stack
custom-setup:
  dependencies:
  - base >=4.10 && < 5
  - Cabal >= 2.4
  - filepath
